src/main/java/org/takes/rs/RsWithBody.java

```diff
@@ -43,6 +46,11 @@
 public final class RsWithBody extends RsWrap {
 
     /**
+     * String for content lenght key:value.
+     */
+    private static final String CONT_LGTH = "Content-Length:%s";
```

Comment:
when I see private static variable my first though is "code might be duplicated".
And yes, do you see that code you've added is almost the same in 3 methods?
Except of how actually content length is calculated.

I propose, instead of ```getHeadAttributes``` create a method:
```java
private static Iterable<String> append(final Response res, final int length) {}
```
which will be responsible by appending content length header to any response, but actual content length will be provided by caller. Thus, we will get rid of code duplication and concentrate content-length update in one place.
Make sense?